src/System.Reflection.DispatchProxy/tests/System.Reflection.DispatchProxy.Tests.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Reflection.DispatchProxy.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
I'm so surprised that this file turned out to be so simple, that I am suspecting I did something wrong 😄.